### ISSUE TITLE
No type checking for React namespaced components

### ISSUE THREAD

**Issue by mhagmajer** [facebook/flow#1113]
Seems like flow completely ignores [Namespaced Components](https://facebook.github.io/react/docs/jsx-in-depth.html#namespaced-components):

``` js
import UI from 'material-ui';

const UIList = UI.List;
<UIList />; // finds error related to missing properties

<UI.List />; // no error
<ThisDoes.Not.Exist />; // no error either
```

**Comment by samwgoldman** [comment]
Thanks for the report! See `TODO` comments in the [relevant code in Type_inference_js](https://github.com/facebook/flow/blob/0a4395c5958e9633db168f68003b09be2630879e/src/typing/type_inference_js.ml#L4242).

**Comment by minedeljkovic** [comment]
This seems to be fixed in 0.53. Is that right @calebmer ?

**Comment by calebmer** [comment]
Yes it has been fixed! This is a cool pattern that we are excited to support 😊
